src/routes/forumComments/getForumCommentsByUserID.js

```diff
@@ -0,0 +1,34 @@
+const express = require('express');
+const router = express.Router();
+const mysql = require('mysql');
+const config = require('../../../config.js');
+
+// Loads the existing comments for a forum
+router.post('/api/getForumCommentsByUserID', (req, res) => {
```

Comment:
to change this method to a `GET` method, look at the following code:
```
router.get('/api/getForumCommentsByUserID', (req, res) => {

    let connection = mysql.createConnection(config);

    let sql = 
    `SELECT forumComments.id, forumComments.forumID, forumComments.userID, forumComments.commentDateTime, forumComments.comment, forums.forumTitle
    FROM shchowdh.forumComments
    LEFT JOIN forums ON forumComments.forumID = forums.id
    WHERE userID = ?;`;
    console.log(sql);
    let data = [req.query.userID];
...
```

This works on Postman:
![image](https://user-images.githubusercontent.com/57375371/225394948-4d0b7759-9bbb-4d94-a0b7-de5f84c11b48.png)

Note: you will need to update any calls to this API to use the `GET` method and pass a query instead of a body.